app/src/main/java/com/nervousfish/nervousfish/activities/ContactActivity.java

```diff
@@ -0,0 +1,50 @@
+package com.nervousfish.nervousfish.activities;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+
+import com.nervousfish.nervousfish.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The ContactActivity shows the contacts information and his public keys.
+ */
+@SuppressWarnings("PMD.AtLeastOneConstructor")
```

Comment:
Maybe it would be better if we add explicit `private` constructors (that are empty) to prevent accidental instantiation of activities. What do you think @jverbraeken ?